examples/distributing/uniffi-rust-to-python-library/run.bat

```diff
@@ -0,0 +1,6 @@
+cargo run --features=uniffi/cli --bin uniffi-bindgen generate .\src\math.udl --language python --out-dir out
```

Comment:
Most of the scripts in this repo assume the python 3 executable is named `python3` (and pip is probably best executed via `python3 -m pip ...`)